### PR TITLE
Strings extracted from file manager template

### PR DESCRIPTION
**Pull request by j-dornbusch**
Extracts and translates all strings from "file manager" template.
English unchanged, Yandex automated translations for other languages, French reviewed.  

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Comment by j-dornbusch** [comment]
@cjcolvar Locally, I have this remaining test failure : 

>  1) Jasmine expects all jasmine tests to pass
>      Failure/Error: expect(jasmine_out).not_to include "\n0 specs"
>        expected "Output: Puma starting in single mode...\n* Version 3.12.0 (ruby 2.4.3-p205), codename: Llamas in Paj... tcp://0.0.0.0:39327\nUse Ctrl-C to stop\njasmine server started\n\n0 specs, 0 failures\n\n Errors:" not to include "\n0 specs"

@jmfrenzel German translations could probably be improved.

File: config/locales/hyrax.de.yml [hunk 1]
```diff
@@ -348,6 +348,15 @@ de:
...
       citations:
         header: 'Zitate:'
+      file_manager:
+        back_to: Zurück zu
+        toolbar: Symbolleiste
+      file_manager_actions:
+        save: Speichern
+        sort_alphabetically: Alphabetisch sortiert
+      file_manager_member_resource_options:
+        representative_media: Vertreter Der Medien
```
Comment (jmfrenzel) [inline config/locales/hyrax.de.yml]: Repräsentative Medien

**Comment by jmfrenzel** [comment]
@j-dornbusch  https://github.com/samvera/hyrax/pull/3679/files#r266859607

**Comment by j-dornbusch** [comment]
@cjcolvar I integrated the change requested by @jmfrenzel , I think it's ready for merge.

**Comment by ShanaLMoore** [comment]
Confirmed translation additions for file manager:
Screen Shot 2019-04-29 at 11 03 11 AM

**Pull request merged**